src/components/IncomingMessageList/ConversationPreviewModal.tsx

```diff
@@ -103,6 +76,11 @@ const ConversationPreviewModal: React.FC<ConversationPreviewModalProps> = (
   } = props;
   const isOpen = conversation !== undefined;
 
+  const { firstName, lastName } = conversation?.contact ?? {};
+  const title = `Conversation Review: ${firstName} ${lastName}`;
+  const { id: campaignId, title: campaignTitle } = conversation?.campaign ?? {};
+  const subheader = `Campaign ${campaignId}: ${campaignTitle}`;
```

Comment:
Suggestion / Question: If these values might not exist, would be a better idea to have a version that works without the values. Especially since the older version supported it.